File: PRAK603_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.c
#include <stdio.h>

int main() {
    int satu, dua;
    scanf("%d %d", &satu, &dua);

    if (satu != dua) {
        printf("Jumlah Tidak Sama\n");
    } else {
        int BariSatu[satu];
        int BarisDua[satu];

        for (int i = 0; i < satu; i++) {
            scanf("%d", &BariSatu[i]);
        }

        for (int i = 0; i < satu; i++) {
            scanf("%d", &BarisDua[i]);
        }

        for (int i = 0; i < satu; i++) {
            printf("%d", BariSatu[i] * BarisDua[i]);
        }
    }

    return 0;
}

File: PRAK603_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.py
satu, dua = map(int, input().split())
if satu != dua:
    print("Jumlah Tidak Sama")
else:
    BariSatu = list(map(int, input().split()))
    BarisDua = list(map(int, input().split()))
    for i in range(satu):
        print(BariSatu[i] * BarisDua[i], end ="")